packages/code-du-travail-data/dataset/glossaire/index.js

```diff
@@ -1,36 +1,22 @@
-const GoogleSpreadsheets = require("google-spreadsheets");
-const { promisify } = require("util");
-const getCells = promisify(GoogleSpreadsheets.cells);
+const fetch = require("node-fetch");
+const slugify = require("@cdt/data/slugify");
 
-const SPREADSHEET_KEY = "1WrmotMiu4kBxRTKW47Q3CzNjxc0GycfAilSHcXH_hfA";
+const DATAFILLER_URL =
+  process.env.DATAFILLER_URL || "https://datafiller.num.social.gouv.fr";
 
-const columns = {
-  term: 2,
-  abbrev: 3,
-  synonym: 4,
-  definition: 6,
-  definition_cdtn: 7
-};
-
-function transformRow(row) {
-  return Object.entries(columns).reduce(
-    (state, [key, colIndex]) => ({
-      ...state,
-      [key]: row.hasOwnProperty(colIndex) ? row[colIndex].value : undefined
-    }),
-    {}
-  );
-}
+const RECORDS_URL = `${DATAFILLER_URL}/kinto/v1/buckets/datasets/collections/glossaire/records`;
 
 async function getGlossaire() {
-  const { cells } = await getCells({
-    key: SPREADSHEET_KEY,
-    worksheet: 1
-  });
-  return Object.values(cells)
-    .slice(1)
-    .filter(row => row.hasOwnProperty("1"))
-    .map(transformRow);
+  const response = await fetch(RECORDS_URL);
+  const items = await response.json();
+  return items.data.map(({ title, abbrs, variants, definition, refs }) => ({
+    title,
+    slug: slugify(title),
+    abbrs,
+    variants,
+    definition,
+    refs
+  }));
```

Comment:
`abbrs` and `variants` are multiline string, we should split on `\n` on fetch ?

Reply:
Alright!  we should translate to an array before json